### PR TITLE
Gitpodify repository

### PR DESCRIPTION
**Pull request by jmMeessen**
PR to add Gitpod configuration. Accommodates Java 17 requirement for VSC Language server

### CODE REVIEW

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-full
+
+USER gitpod
+
+RUN bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && \
+    sdk install java 17.0.3-tem && \
```
Comment (jglick) [inline .gitpod.Dockerfile:6]: Does not seem right to hard-code a patch release here, in a way opaque to Dependabot.

Comment (jmMeessen) [reply .gitpod.Dockerfile:6]: Good point.
Any idea to explore in order to fix/improve this ?

Comment (jglick) [reply .gitpod.Dockerfile:6]: Sorry, not offhand.

Comment (jmMeessen) [reply .gitpod.Dockerfile:6]: Thank you.
I will run this problem with the people from Infra and see if they have an idea. 

In the mean time, I propose to  flag this as a technical debt.

**Pull request merged**